src/node/wrapper.ts

```diff
@@ -71,6 +65,19 @@ export class IpcMain {
     }
   }
 
+  /**
+   * Ensure we control when the process exits.
+   */
+  public preventExit(): void {
```

Comment:
Ahh this makes way more sense now, we're only doing it in the child right?

Reply:
Yup! We don't run any 3rd party code in the parent so we don't need it there.

Reply:
Yea we were overriding it before which was causing the `process.exit`'s caused by the cloud agent exiting to not `process.exit` the main process.

Reply:
Ahhh that makes sense.